source/reference/beta-program.txt

```diff
@@ -23,25 +23,36 @@ Get Started
 To request access to the {+c2c-beta-program-short+} binaries, please file a 
 `HELP ticket <https://jira.mongodb.org/browse/HELP>`_.
 
-.. warning:: Disclaimer
-
-   MongoDB restricts access to beta features via the beta build. Your use of the 
-   beta is governed by the language specified in the 
-   `Cloud Subscription Agreement <https://www.mongodb.com/cloud-subscription-agreement/november-2023>`_, 
-   `Cloud Terms of Service <https://www.mongodb.com/legal/terms-and-conditions/cloud>`_, 
-   or other applicable agreement between you and MongoDB with respect to your 
-   use of ``mongosync``.
+When you first run the {+c2c-beta-program-short+} binary, you must accept the 
+following disclaimer:
+
+.. code-block:: shell
+   :copyable: false
+  
+   MongoDB restricts access to beta features via the beta build. Your use 
+   of the beta is governed by the language specified in the Cloud 
+   Subscription Agreement, Cloud Terms of Service, or other applicable 
+   agreement between you and MongoDB with respect to your use of 
+   ``mongosync``.
```

Comment:
Question: Does the shell output include the double ticks?  I imagine it shouldn't but we should follow whatever the user sees in the output.

Reply:
Ah that was left over from when this used to be an admonition instead of a code block. Updating.